.travis.yml

```diff
@@ -23,8 +23,10 @@ before_install:
   - docker ps -a
 
 install:
-  - go get -u github.com/pilosa/go-pilosa
-  - cd "$GOPATH/src/github.com/pilosa/go-pilosa" && git checkout v0.9.0 && cd "$TRAVIS_BUILD_DIR"
+  - go get -u github.com/golang/dep/cmd/dep
+  - cd "$TRAVIS_BUILD_DIR"
```

Comment:
i think we no longer need this

Reply:
you mean `cd "$TRAVIS_BUILD_DIR"`? So far I left it just in case.

Reply:
yep. it was because it used to CD into go-pilosa's dir. Now we don't need that anymore